csharp/Test.cmd

```diff
@@ -5,19 +5,33 @@ SET CMDHOME=%~dp0
 @REM Remove trailing backslash \
 set CMDHOME=%CMDHOME:~0,-1%
 
-SET VSIDEDIR=%VS120COMNTOOLS%..\IDE
-SET VSTESTEXEDIR=%VSIDEDIR%\CommonExtensions\Microsoft\TestWindow
-SET VSTESTEXE=%VSTESTEXEDIR%\VSTest.console.exe
+if "%NUNITCONSOLE%" == "" (
+  GOTO :NUNITCONSOLEERROR
+)
 
 cd "%CMDHOME%"
 @cd
 
-SET CONFIGURATION=Release
+SET CONFIGURATION=Debug
```

Comment:
We have mock tests in unit tests, which rely on the assumption that the stack trace will be exactly the same as the code is written. But in Release build, optimizations will inline those methods, which makes the tests fail (or at least unreliable).